### ISSUE TITLE
Relative paths in config file are un-usefully relative to whatever directory is current when doxygen is run (Origin: bugzilla #756161)

### ISSUE THREAD

**Issue by doxygen** [doxygen/doxygen#5881]
status NEW severity *enhancement* in component *general* for *---*
Reported in version *1.8.9.1* on platform *Other*
Assigned to: Dimitri van Heesch

On 2015-10-07 01:31:49 +0000, gnome.org@erewhon0.net wrote:

> Relative paths given in a doxygen config file, e.g., those set on INPUT, are relative to the current directory when Doxygen is run.
> 
> This means that the information about the directory in which to run doxygen must be recorded somewhere, either in one's head, in a note, or in an external build system.
> 
> I think it would be better if paths were relative to the config file location, then no other information would be needed that could be forgotten. The command could be run anywhere and it would work.
> 
> I imagine such a change might break a few projects so I propose adding a new config entry at the top of the file to select the behavior for relative paths. This would default to the current behavior.

On 2016-03-31 14:28:34 +0000, Fabrizio Da Ros wrote:

> I fully agree. It makes much more sense to have paths relative to the doxyfile location. Actually I thought it was like that since I run doxygen through a script from the project root where the doxyfile is stored too.

**Comment by Matze693** [comment]
I agree, too. Relative paths would be fine. For example, especially for the tag `HHC_LOCATION`. Our doxyfile is located in a Git-Repo. Due to the fact, that a user can clone a repo wherever he want, absolute paths are a bad solution.

Is a solution for this issue planned?
